R/circropper.R

```diff
@@ -5,27 +5,40 @@ library(magick)
 #' Create a new image cropped by a circle mask and leave a margin
 #'
 #' @param input_path character the image file path
-#' @param outut_path character the saved image path
-#' @param margin double
+#' @param margin double the distance between circle boundary and the original image boundary
 #'
-#' @return an image file saved in the designated folder
+#' @return a cropped image
 #' @export
+#'
 #' @examples
-#' circropper("path_to_input_img.png", "output_img.png", 20)
+#' \dontrun{circropper("bear.jpg", 20)}
+
+circropper <- function(input_path, margin) {
+  # Check argument type
+  if (!is.character(input_path) & !is.double(margin)){
+    stop("TypeError: The 'input_path' arguments must be a character and the 'margin' argument must be a double")}
```

Comment:
I would remove this first exception case as it is redundant.
```suggestion
```

Reply:
The same reason as in `pyimage`. I think it's not redundant. I want to notify the user that he has two wrong arguments at the same time.

Reply:
I believe it is standard for functions to only report input type errors one at a time; however, if you think this is beneficial then we can keep it :)